### PR TITLE
✨ NeoDropdown & NeoSelect

### LINKED ISSUE #4361: create NeoDropdown component based on o-dropdown in histoire
from 
- #4354

### LINKED ISSUE #4363: Create NeoSelect
While reviewing #4297 I realized we do not have any common select :)

ref:
- #4354

### PR DESCRIPTION
**Pull request by roiLeo**
#### // TODO:

- [x] dark-mode for NeoSelect & NeoDropdown
- [ ] font-awesome integration in histoire
- [x] check oruga scss global var
- [ ] `active` style
- [x] new props/slot
- [ ] responsive dropdown (mobile version)

## PR Type

- [x] Feature

## Context

- [x] closes #4361 
- [x] closes #4363 
- [x]  fix IdentityItem import
- [x] `<NeoSelect />` component on histoire
- [x] `<NeoDropdown />` component on histoire

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

## Screenshot 📸

Screenshot 2022-11-30 at 14-27-54 NeoDropdown › With Button Histoire

### CODE REVIEW

**Comment by yangwao** [comment]
Awesome stuff! 😍

Is it mergeable already?

**Comment by roiLeo** [comment]
> Awesome stuff! 😍
> 
> Is it mergeable already?

ještě ne
Screenshot 2022-12-08 at 12-48-42 KodaDot - Kusama NFT Market Explorer

**Comment by roiLeo** [comment]
This PR break current oruga tabs design, I'll be looking why 

Screenshot 2022-12-08 at 14-56-59 KodaDot - Kusama NFT Market Explorer

**Comment by roiLeo** [comment]
> i see, what hapeen if we try with sass variables? does the sass variable also get the same result?

Sass variable ain't working :/

**Comment by roiLeo** [comment]
Thanks for your help! I'm not really a fan of the current solution, but it look like it's working (don't work when using `scoped` scss in `NeoDropdown` component)

File: libs/ui/src/components/NeoDropdown/NeoDropdown.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+  <o-dropdown
+    aria-role="list"
+    :position="position"
+    :append-to-body="appendToBody">
+    <template #trigger="{}">
```
[the remark this answers is not in the document]
Comment (roiLeo) [reply libs/ui/src/components/NeoDropdown/NeoDropdown.vue]: false alarm but it's fixed

**Comment by yangwao** [comment]
>I'm not really a fan of the current solution

what would be better solution?

**Comment by yangwao** [comment]
oh okay, for now we can merge then?

**Comment by roiLeo** [comment]
> oh okay, for now we can merge then?

I think it's good to go, don't mind failing test

**Pull request merged**